docs/worker-node/install-singularity.md

```diff
@@ -49,23 +50,27 @@ As with all OSG software installations, there are some one-time (per host) steps
 - Ensure the host has [a supported operating system](/release/supported_platforms)
 - Obtain root access to the host
 - Prepare the [required Yum repositories](/common/yum)
-- A [CVMFS installation](/worker-node/install-cvmfs) for singularity image distribution
+
+In addition, with Singularity this is highly recommended:
```

Comment:
1. You don't need the "with Singularity" b/c the whole doc is about Singularity
2. Why is CVMFS highly recommended? Is it still for image distribution? Maybe you should add that part back.

Reply:
I will put it in the introductory part.  It didn't make sense in the instruction part, the bullet point.